Misc/NEWS.d/next/Security/2024-12-05-21-35-19.gh-issue-127655.xpPoOf.rst

```diff
@@ -0,0 +1 @@
+Fixed the :class:`asyncio.SelectorEventLoop` transport not pausing writes for the protocol when the buffer reaches the high water mark when using :meth:`asyncio.WriteTransport.writelines`.
```

Comment:
I suggest to use the actual name `_SelectorSocketTransport`, the current wording sounds like it would covers the SSL impl too which it isn't.